redfish-core/lib/memory.hpp

```diff
@@ -791,10 +791,11 @@ inline void requestRoutesMemoryCollection(App& app)
         asyncResp->res.jsonValue["Name"] = "Memory Module Collection";
         asyncResp->res.jsonValue["@odata.id"] =
             "/redfish/v1/Systems/system/Memory";
-
```

Comment:
Looking at https://gerrit.openbmc.org/c/openbmc/bmcweb/+/59537/12/redfish-core/lib/memory.hpp#714, there seems another change is at getDimmData() to use getSubTree().

Reply:
I made changes to the code in ibm-openbmc/bmcweb not upstream

Reply:
What I meant is that the change in ibm-openbmc/bmcweb seems not correct.
upstream link is a reference to compare

Reply:
Do we have that code in memory.hpp?

Reply:
Please put a reference to ibm-openbmc/bmcweb

Reply:
Similar to the other case, gerrit changes the code in the function of getDimmData() to use getSubTree().
However, it is not in ibm-openbmc/bmcweb.

https://gerrit.openbmc.org/c/openbmc/bmcweb/+/59537/12/redfish-core/lib/memory.hpp#714

Current upstream memory.hpp:

```
706 inline void getDimmData(std::shared_ptr<bmcweb::AsyncResp> aResp,
707                         const std::string& dimmId)
708 {
709     BMCWEB_LOG_DEBUG << "Get available system dimm resources.";
710+    constexpr std::array<std::string_view, 2> dimmInterfaces = {
711 +         "xyz.openbmc_project.Inventory.Item.Dimm",
712+          "xyz.openbmc_project.Inventory.Item.PersistentMemory.Partition"};
713 +     dbus::utility::getSubTree(
714         "/xyz/openbmc_project/inventory", 0, dimmInterfaces,
715         [dimmId, aResp{std::move(aResp)}](
716             const boost::system::error_code& ec,
717             const dbus::utility::MapperGetSubTreeResponse& subtree) {
```